pkg/losetup/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM linuxkit/alpine:3683c9a66cd4da40bd7d6c7da599b2dcd738b559 AS mirror
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out  \
+    alpine-baselayout \
+    busybox
+
+# Remove apk residuals
+RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
+
+FROM scratch
+ENTRYPOINT []
+WORKDIR /
+COPY --from=mirror /out/ /
+
+COPY loopy.sh /usr/bin/loopy
```

Comment:
this also has the `loopy.sh` -> `loopy` rename

Reply:
same as per https://github.com/linuxkit/linuxkit/pull/3249#discussion_r243295417

Reply:
Yes again. If @rn is ok with it, let's just name it `loopy` in the src and be consistent. I would say that is what we do elsewhere, but I think almost all of the `sh` code is replaced by `go`-sourced binaries by now. :-)